### PR TITLE
Fix vulnerability caused by filename escaping

### PR DESCRIPTION
**Pull request by motoyasu-saburi**
Detail:
https://github.com/kittinunf/fuel/issues/876

~Local tests have not yet been run.~
~(I need to set up the Android-SDK perimeter, which is taking a bit of time.)~

### CODE REVIEW

**Comment by kittinunf** [comment]
Hey thanks so much for the fix! I will take a look on this tonight.

File: fuel/src/test/kotlin/com/github/kittinunf/fuel/core/DataPartTest.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        assertThat(
+                "ContentDisposition filename must escape Double-Quote and CRLF",
+                specialCharFilePart.contentDisposition == "form-data; name=\"second\"; filename=\"$expectEscapedFilename\""
+        )
+        assertThat(
+                "ContentDisposition filename must escape Double-Quote and CRLF",
+                specialCharBlobPart.contentDisposition == "form-data; name=\"third\"; filename=\"$expectEscapedFilename\""
+        )
+
+        val normalFileName = "abc.txt"
```
Comment (kittinunf) [inline fuel/src/test/kotlin/com/github/kittinunf/fuel/core/DataPartTest.kt:34]: Could you please split this into another test function? This is to make sure that we have a small test case function testing only 1 thing.

**Review by kittinunf — APPROVED** [review]
Small request, otherwise this looks good 👍

**Comment by motoyasu-saburi** [comment]
@kittinunf Thank you for your review 🙇 
Could you review it again?

**Review by kittinunf — APPROVED** [review]

**Comment by kittinunf** [comment]
@motoyasu-saburi I will release the patch for this soon please hang on tight.

**Comment by Shpota** [comment]
@kittinunf thank you for the library. When are you planing to push the v2 release?

**Pull request merged**